### PR TITLE
ES and TH dashboards

### PR DESCRIPTION
**Pull request by awisema**

### CODE REVIEW

File: app/models/warehouse_report/outcomes/es_report.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  def columns_for_percent_in_stabilization
+    {
+      service_project: _('Housing Search'),
+      search_start: _('Search Start'),
+      search_end: _('Search End'),
+      residential_project: _('Stabilization Project'),
+      housed_date: _('Date Housed'),
+      housing_exit: _('Housing Exit'),
+    }
+  end
```
Comment (eanders) [inline app/models/warehouse_report/outcomes/es_report.rb]: These translations don't feel right for ES.  Are they actually used?

Comment (awisema) [reply app/models/warehouse_report/outcomes/es_report.rb]: Good question -- I don't think so -- so, they shouldn't be there.

File: app/models/warehouse_report/outcomes/th_report.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  def columns_for_percent_in_stabilization
+    {
+      service_project: _('Housing Search'),
+      search_start: _('Search Start'),
+      search_end: _('Search End'),
+      residential_project: _('Stabilization Project'),
+      housed_date: _('Date Housed'),
+      housing_exit: _('Housing Exit'),
+    }
+  end
```
Comment (eanders) [inline app/models/warehouse_report/outcomes/th_report.rb]: Same here as with ES.  Maybe these should exist in `WarehouseReport::Outcomes::Base`?

File: app/views/warehouse_reports/outcomes/es_report/_index.haml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    var destination_chart = new App.WarehouseReports.Rrh.Destination('.rrh__destination-chart', '.rrh__destination-chart-legend', destination_chart_data, '#{clients_warehouse_reports_rrh_index_path(passed_params.merge(metric: :destination))}');
+    var ph_returns_data = #{@report.ph_returns_for_chart.to_json.html_safe}
+    var ph_returns_chart = new App.WarehouseReports.Rrh.Returns('#jPhReturnsChart', ph_returns_data, '#{clients_warehouse_reports_rrh_index_path(passed_params.merge(metric: :return_after_exit_to_ph))}');
+    var returns_data = #{@report.returns_for_chart.to_json.html_safe}
+    var returns_chart = new App.WarehouseReports.Rrh.Returns('#jReturnsChart', returns_data, '#{clients_warehouse_reports_rrh_index_path(passed_params.merge(metric: :return_after_exit_to_any))}');
+    var percent_exiting_stabilization_data = #{@report.percent_exiting_stabilization_data.to_json.html_safe}
+    var percent_exiting_stabilization_chart = new App.WarehouseReports.Rrh.Makeup('.percentExitingStabilizationChart', percent_exiting_stabilization_data, '#{clients_warehouse_reports_rrh_index_path(passed_params.merge(metric: :percent_exiting_stabilization))}')
```
Comment (eanders) [inline app/views/warehouse_reports/outcomes/es_report/_index.haml]: The paths in here and for TH, maybe PSH?  All need adjustments, they are loading the `RRH` data.

**Review by eanders — CHANGES_REQUESTED** [review]
Counts between the charts and drill-downs don't match
Screen Shot 2021-01-22 at 1 47 39 PM
Screen Shot 2021-01-22 at 1 47 47 PM

Also, on the TH dashboard, the link to `Clients Exited` is pointing to shelter....
Screen Shot 2021-01-22 at 1 49 16 PM

**Pull request merged**